meetings/Agenda_2018-02-20.md

```diff
@@ -0,0 +1,19 @@
+## Meeting Agenda, February 20, 2018
+
+1. **RQM course, next round (notes [here](https://github.com/UofTCoders/council/pull/221))**
+ * What teaching structure do we want? 4 TAs, 2 instructors and 2 TAs? Can guest lecturers be paid? 
+ * What material should we add or remove in the analysis techniques section? (Related to who gets to be head instructor)
+ * Writing a paper? Should we start taking a crack at this?
```

Comment:
I'm planning on starting something on this over this weekend, so I can take the first crack at it. Let me know what you guys talk about tho for it.

Reply:
PS I miss everyone!